### PR TITLE
Update _nfs.rst

### PR DESCRIPTION
**Pull request by YangShizhao**
fix the wrong command of mount folder to /primary and /secondary;
add new command to check if the service is running;
add comments to explain what to do if no file exists;

### CODE REVIEW

File: source/installguide/management-server/_nfs.rst [hunk 1]
```diff
@@ -246,15 +246,21 @@ operating system version.
          chkconfig rpcbind on
          reboot
 
+   #. The command chkconfig is no longer available in Ubuntu, to check if the nfs service has been running, use following command, if you see words like *nfsd*, the service has been started succcessfully:
```
Comment (DaanHoogland) [inline source/installguide/management-server/_nfs.rst:249]:
```suggestion
   #. The command chkconfig is no longer available on Ubuntu systems. To check if the nfs service has been running, use following command, if you see words like *nfsd*, the service has been started successfully:
```
I also sugest adding the version of ubuntu from which it is no longer available

**Review by DaanHoogland — APPROVED** [review]
thanks @YangShizhao , I have some small remarks.

**Comment by shwstppr** [comment]
@YangShizhao can you please check the review comments?

**Comment by YangShizhao** [comment]
fixed

**Pull request closed without merging**